operator/src/main/java/oracle/kubernetes/operator/helpers/JobHelper.java

```diff
@@ -406,4 +415,68 @@ public NextAction onSuccess(Packet packet, CallResponse<V1PodList> callResponse)
       return doNext(packet);
     }
   }
+
+  // Convert a job log message to an operator log message
+  //   - assumes the messages contins a string like '[SEVERE]', etc, if
+  //     not, then assumes the log level is 'FINE'.
+  private static void logToOperator(StringBuffer logStr, StringBuffer extraStr) {
+    String logMsg = "Introspector Job Log: " 
```

Comment:
i18n.  Can this please come from the resource bundle?

Reply:
I'm thinking this should be a string that never changes and is easy to grep out of the log - so it's not a straightforward i18n candidate IMO.   Can you suggest something that can be hard-coded?   How about "[INTROSPECTORJOB] "?

Reply:
@rjeberhard